### PR TITLE
Replace molecules with packages/ui

### LINKED ISSUE #1028: [fe][wiki] Remove molecules, replace with packages/ui

### PR DESCRIPTION
**Pull request by marmorrei**
Closes #1028 
Only missing to replace TopicsEditableItem with EditableItem that is being done in a separate issue https://github.com/IT-Academy-BCN/ita-wiki/issues/1053

### CODE REVIEW

**Review by kevinmamaqi — APPROVED** [review]

File: web/wiki/src/__tests__/organisms/CardResource.test.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { vi } from 'vitest'
 import { render, screen } from '../test-utils'
-import CardResource from '../../components/organisms/CardResource'
+import { CardResource } from '../../components/organisms'
 import { TAuthContext, useAuth } from '../../context/AuthProvider'
 import { TCardResource } from '../../types'
```
Comment (NataliaR-BCN) [inline web/wiki/src/__tests__/organisms/CardResource.test.tsx:6]: Please, include tests for new code from VoteCounter

File: web/wiki/src/components/organisms/Navbar.tsx [hunk 2]
```diff
@@ -162,6 +162,8 @@ export const Navbar = ({
         <SelectLanguage />
```
Comment (NataliaR-BCN) [inline web/wiki/src/components/organisms/Navbar.tsx:162]: This needs a component?

File: web/wiki/src/components/organisms/Navbar.tsx [hunk 3]
```diff
@@ -173,7 +175,11 @@ export const Navbar = ({
         ) : null}
 
         <UserButton />
```
Comment (NataliaR-BCN) [inline web/wiki/src/components/organisms/Navbar.tsx:177]: This needs a component?

File: web/wiki/src/__tests__/organisms/CardResource.test.tsx (hunk 1 above)
Comment (marmorrei) [reply web/wiki/src/__tests__/organisms/CardResource.test.tsx:6]: It will be addressed in issue #1060

**Review by NataliaR-BCN — APPROVED** [review]

**Review by kevinmamaqi — APPROVED** [review]

**Pull request merged**